repos/sourcelink.proj

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PreferredHash>51310e65e75010467993f793e1739d1a1dad50c5</PreferredHash>
+    <SourceDirectory>sourcelink</SourceDirectory>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+
+  <PropertyGroup>
+    <OfficialBuildId>20190623.1</OfficialBuildId>
+
+    <BuildCommandArgs/>
+    <BuildCommandArgs>$(BuildCommandArgs) --pack</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) --configuration $(Configuration)</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) --binaryLog</BuildCommandArgs>
+    <BuildCommand>$(ProjectDirectory)build$(ShellExtension) $(BuildCommandArgs)</BuildCommand>
+
+    <GlobalJsonFile>$(ProjectDirectory)global.json</GlobalJsonFile>
+    <NuGetConfigFile>$(ProjectDirectory)/NuGet.config</NuGetConfigFile>
```

Comment:
Redundant `/`? (By convention, MSBuild directory properties always already have a trailing directory separator.)

Reply:
I think I got this from a merge update from the weekly sb update. I'll go ahead and fix this one.

Reply:
Still have one extra slash here.